style.css

```diff
@@ -525,7 +538,7 @@ textarea {
 
   .navbar-links2 {
     display: flex;
-    margin-right: 3rem;
+    margin-right: 12rem;
```

Comment:
- [ ] Very good job in implementing the changes requested by the previous reviewer, however, as mentioned by the Previous Reviewer, by using `6rem` instead of `12rem` will make your design much more similar to the given design, Please resolve it. The Figma design serves as a visual reference and guideline for implementing the desired user interface and overall look of your application.  matching your design to the given design requirements enhances the chances of creating a successful and well-rounded design solution that meets the needs of users, complies with regulations, and performs effectively and efficiently. :slightly_smiling_face: 

The example below shows the difference between yours and the accepted one: :arrow_down: 

 <table>
    <tr>
      <td>Yours </td>
      <td>Accepted type</td>
    </tr>
    <tr>
      <td>

 
![Student Desktop](https://github.com/denigit5/Portfolio/assets/43528594/7a702485-64d5-437e-bc69-18f94d4b6379)

 
</td>
      <td>

![Figma Home Page](https://github.com/denigit5/Portfolio/assets/43528594/7f719179-1f73-4bac-82a4-1b16dcd13e03)

</td>
    </tr>
  </table>